rewrite-maven/src/test/java/org/openrewrite/maven/AddAnnotationProcessorTest.java

```diff
@@ -245,20 +230,50 @@ void addAnnotationWithVersionAsMavenProperty() {
                               <artifactId>maven-compiler-plugin</artifactId>
                               <configuration>
                                   <annotationProcessorPaths>
-                                      <path>
-                                          <groupId>org.mapstruct</groupId>
-                                          <artifactId>mapstruct-processor</artifactId>
-                                          <version>${version.mapstruct}</version>
-                                      </path>
                                       <path>
                                           <groupId>org.projectlombok</groupId>
-                                          <artifactId>lombok</artifactId>
-                                          <version>${version.lombok}</version>
+                                          <artifactId>lombok-mapstruct-binding</artifactId>
+                                          <version>0.2.0</version>
```

Comment:
This is not what we want, right?

Reply:
Indeed better to update the version tag; this was a first quick attempt at getting rid of the comparison failure. Improvement picked up in https://github.com/openrewrite/rewrite/pull/4822/commits/7c2fcd29d3a3a973e7b33a59263f4819d7807de0